flocker/sample/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
```

Comment:
This won't be in the final code for sure, and it probably won't be this but moved. If this design is deemed appropriate then there can be tests for the test tools which do something similar to eyeballing the results of these tests with different variables supplied.